src/x86_64-fma/fft16x16.py

```diff
@@ -499,6 +500,10 @@ def inverse_vfft(reg_t0, reg_t8, reg_t_stride, data_in, reg_row_start=None, reg_
                     negate_b=fft8_negate_b.get(id(data_hi), False),
                     writeback=False)
 
+            if relu:
+                ymm_zero = YMMRegister()
+                VMOVAPS(ymm_zero, Constant.uint32x8(0))
```

Comment:
Could you please tell me the reason to use negative signed zero? Or propose a simple example?